### PR TITLE
es6 imports / commonjs require

### PR DESCRIPTION
**Pull request by SeyedMojtaba1**
related #5

### CODE REVIEW

File: Seyed/commonjs/ESA/package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "module": "^1.2.5"
```
Comment (mkermani144) [inline Seyed/commonjs/ESA/package.json:3]: What's this? Probably a mistake? 🤔

File: Seyed/commonjs/ESA/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (mkermani144) [inline Seyed/commonjs/ESA/.gitignore:1]: Please note that it's not _ESA_, but _ESM_ (or **E**CMA**S**cript **M**odules).

File: Seyed/commonjs/commonjs/isNil.cjs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// isNil.js
+function isNil (value) {
+    return value == null;
+}
+
+module.exports = {isNil};
```
Comment (mkermani144) [inline Seyed/commonjs/commonjs/isNil.cjs:6]: While default and named exports are mostly ESM concepts, this is not a so called default export. You can directly export the `isNil` itself without wrapping it in an object.

File: Seyed/commonjs/commonjs/package.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "module": "^1.2.5"
```
Comment (mkermani144) [inline Seyed/commonjs/commonjs/package.json:3]: Same here.

**Review by mkermani144 — CHANGES_REQUESTED** [review]

File: Seyed/commonjs/ESA/package.json (hunk 1 above)
Comment (SeyedMojtaba1) [reply Seyed/commonjs/ESA/package.json]: I fixed it. It's my mistake.

File: Seyed/commonjs/ESA/.gitignore (hunk 2 above)
Comment (SeyedMojtaba1) [reply Seyed/commonjs/ESA/.gitignore:1]: Sure

File: Seyed/commonjs/commonjs/isNil.cjs (hunk 3 above)
Comment (SeyedMojtaba1) [reply Seyed/commonjs/commonjs/isNil.cjs:6]: Okay, I understood.

File: Seyed/commonjs/commonjs/package.json (hunk 4 above)
Comment (SeyedMojtaba1) [reply Seyed/commonjs/commonjs/package.json:3]: I fixed it.

**Review by mkermani144 — APPROVED** [review]

**Pull request merged**